packages/web/src/components/createCell.tsx

```diff
@@ -375,12 +390,19 @@ export function createCell<
             {...props}
             {...afterQueryData}
             updating={loading}
-            {...queryRest}
+            queryResult={queryResult}
           />
         )
       }
     } else if (loading) {
-      return <Loading {...{ ...queryRest, ...props }} />
+      return (
+        <Loading
+          {...{
+            queryResult,
+            ...props,
+          }}
+        />
```

Comment:
Since we're already introducing all kinds of breaking changes in this PR 😄 I'd change this one to be more inline with the others

```suggestion
        <Loading
          {...props}
          queryResult={queryResult}
        />
```

Reply:
Just for my sanity, this isn't actually breaking right? (All for the change though!)

Reply:
Ah I guess it's the switching them around part that's breaking, got it

Reply:
😆 honestly this went right above me. thanks for the catch @Tobbe 

fixed it